test/components/field-key/revoke.spec.js

```diff
@@ -23,8 +23,6 @@ describe('FieldKeyRevoke', () => {
       const app = await load('/projects/1/app-users');
       const li = app.get('.field-key-row .dropdown-menu li');
       li.classes('disabled').should.be.true();
-      await li.get('a').trigger('click');
-      app.getComponent(FieldKeyRevoke).props().state.should.be.false();
```

Comment:
We no longer need to test this, and in general going forward, trying to test this sort of thing would be a little difficult. Once we verify that `li` has the `disabled` class, we know from the tests of `useDisabled()` that clicking the link will have no effect. The root `App` component calls `useDisabled()`, but most tests don't mount `App` — so `useDisabled()` won't be called in most tests. `useDisabled()` also requires that the component be attached to the document body. Altogether, this means that testing that a `.disabled` link is actually disabled is more difficult. I think it's OK to remove this assertion though, because we have good test coverage of `useDisabled()`.